iron-doc-behavior.html

```diff
@@ -151,10 +170,16 @@
       },
 
       _compareElements: function(a, b) {
-        if (a.tagname === b.tagname) {
-          return a.classname == null ? -1 : a.classname.localeCompare(b.classname);
+        if (a.name && b.name && a.name !== b.name) {
```

Comment:
Before falling back on tagname and classname, we need to handle the cases where one item has a name and the other doesn't